CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Mapbox welcomes participation and contributions from everyone.
 ### Changelog
 [Changes between v2.7.2 and v2.7.3](https://github.com/mapbox/mapbox-navigation-android/compare/v2.7.2...v2.7.3)
 
+#### Bug fixes and improvements
+- Fixed some internal Copilot-related issues and added minor improvements.
```

Comment:
PR link?

Reply:
There were some but opted to not link as they were internal.